bentoml/utils/__init__.py

```diff
@@ -75,3 +79,21 @@ def ProtoMessageToDict(protobuf_msg, **kwargs):
         kwargs['preserving_proto_field_name'] = True
 
     return MessageToDict(protobuf_msg, **kwargs)
+
+
+def _is_pypi_release():
+    is_installed_package = hasattr(version_mod, 'version_json')
+    is_tagged = not BENTOML_VERSION.startswith('0+untagged')
+    is_clean = not version_mod.get_versions()['dirty']
+    return is_installed_package and is_tagged and is_clean
+
+
+def _is_bentoml_in_editor_mode():
```

Comment:
_is_bentoml_in_develop_mode


add a comment: `this is for BentoML developer to create BentoService containing custom develop branches of BentoML library, it gets triggered when BentoML module is installed via "pip install --editable ."`